index.html

```diff
@@ -193,20 +194,27 @@ <h2 class="conta">CONTACT US</h2>
     if you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
 </p>
 
-<form >
-    <input class="text-inp" type="text" placeholder=" Text Input">
-
-    <input class="email" type="text" placeholder=" Jeremy@gmail.com">
-
-    <textarea class="send-text" cols="30" rows="10" placeholder="
-Enter text here"></textarea>
-</form>
-
-<button type="button" class="btn-sec"> See Project </button>
-
-<div class="cont-line">.</div>
-
-
+    <form action="https://formspree.io/f/mlezygyr" method="post">
+        <ul class="input-list">
+         <li>
+           <label for="name">Name:</label>
+           <input type="text" id="name" name="user_name" placeholder="  Name" maxlength="30" required>
+         </li>
+         <li>
+           <label for="mail">E-mail:</label>
+           <input type="email" id="mail" name="user_email" placeholder="  E-mail" required>
+         </li>
+         <li>
+           <label for="msg">Message:</label>
+           <textarea cols="33" rows="16"  id="msg" name="user_message" 
+           placeholder=" 
+ Enter text here" maxlength="500" required ></textarea>
+         </li>
+         <li class="button">
+           <button type="submit" id="bttn"> see project </button>
```

Comment:
- [ ] In the mobile version of the Figma design, the button is aligned horizontally center. please rectify
- [ ] Kindly use the font color as in the Figma design as well.
- [ ] Kindly add the `<hr>' horizontal line at the bottom as in the design.
#### Design
![image](https://user-images.githubusercontent.com/44057863/177428228-eec3b1a8-7fe6-40f8-86f4-816134c4210a.png)

#### Impl
![image](https://user-images.githubusercontent.com/44057863/177428193-0f89eab5-4795-4d9c-b55e-014208d2d198.png)